spec/support/email_alert_api_helpers.rb

```diff
@@ -0,0 +1,16 @@
+require "services"
+require "gds_api/test_helpers/email_alert_api"
+
+module EmailAlertApiHelpers
+  include GdsApi::TestHelpers::EmailAlertApi
+
+  # TODO: Move this to gds-api-adapters
```

Comment:
Is this recorded anywhere else?

Reply:
Good point! I'll add a card to the backlog to do it! There are also 2 other test helpers to move:  https://github.com/alphagov/collections-publisher/search?q=TODO%3A+Move+this+to+gds-api-adapters